crates/stackable-operator/src/commons/networking.rs

```diff
@@ -1,44 +1,136 @@
-use std::{fmt::Display, ops::Deref};
+use std::{fmt::Display, net::IpAddr, ops::Deref, str::FromStr};
 
 use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
+use snafu::Snafu;
 
 use crate::validation;
 
-/// A validated hostname type, for use in CRDs.
-#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+/// A validated domain name type conforming to RFC 1123, e.g. an IPv4, but not an IPv6 address.
```

Comment:
note: This doc comments seems outdated, rewording needed.

Reply:
Ohm I fail to see the outdated parts, can you please make a suggestion?